src/metrics.py

```diff
@@ -40,6 +40,8 @@ def push_image_metrics(self, image):
             "device_type": image.device_type,
             "python_version": image.python_version,
             "image_type": image.image_type,
+            "image_stage": image.stage,
+            "push_required": str(image.to_push)
```

Comment:
metrics of only the images that are qualified for a push should be uploaded? In that case what's the usecase for `push_required`?

Reply:
The images that are qualified for push are just the Common Stage images. The time to process the Common Stage images is not same as the time taken to process Pre_push images. In fact, the time taken to process CommonStage images is quite small as opposed to Pre_push images. Therefore, I thought of segregating the buckets by adding extra dimension so that we can monitor things correctly.

Reply:
Done